applications/nrf_desktop/src/events/hid_report_provider_event.h

```diff
@@ -36,11 +43,13 @@ struct subscriber_conn_state {
 struct hid_report_provider_api {
 	/** @brief Send report to the currently connected subscriber.
 	 *
-	 * The HID provider should send HID report and return true only if HID report data requires
-	 * update (there is user input waiting). Otherwise the HID provider should skip sending HID
-	 * report and return false unless force flag is set. If the force flag is set, HID provider
-	 * needs to always send a HID report (even if the report would contain no new information)
-	 * and return true.
+	 * The HID provider should send HID report as @ref hid_report_event and return true only if
+	 * HID report data requires update (there is user input waiting). Otherwise the HID provider
+	 * should skip sending HID report and return false unless force flag is set.
```

Comment:
```suggestion
	 * The HID provider should send the HID report as a @ref hid_report_event and return true only if
	 * the HID report data requires an update (there is user input waiting). Otherwise, the HID provider
	 * should skip sending the HID report and return false unless a force flag is set.
```